include/swift/AST/DiagnosticsSema.def

```diff
@@ -81,6 +81,10 @@ ERROR(could_not_find_value_member,none,
 ERROR(could_not_find_value_member_corrected,none,
       "value of type %0 has no member %1; did you mean %2?",
       (Type, DeclName, DeclName))
+ERROR(could_not_find_value_dynamic_member_corrected,none,
+      "value of type %0 and dynamic keyPath of type %1 has no member %2; did you mean %3?",
```

Comment:
Sorry, I don't understand this message as it is currently worded; for one, the verb and noun do not agree, so I can't parse the sentence. (Also, the name of the feature is "key path.")

Reply:
Yeah, now that I'm thinking is indeed not clear. The intention here is saying that neither the wrapper type and the dynamic target has a member with that name

Reply:
I've tried to metion both types and follow the same pattern of "value of type ... has not member ..."